pom.xml

```diff
@@ -40,9 +103,9 @@
 			<version>3.3.1-SNAPSHOT</version>
 		</dependency>
 		<dependency>
-			<groupId>com.sk89q</groupId>
-			<artifactId>worldedit</artifactId>
-			<version>6.0.0-SNAPSHOT</version>
+			<groupId>com.sk89q.worldedit</groupId>
+			<artifactId>worldedit-bukkit</artifactId>
```

Comment:
Ah, I see. In that case it should be worldedit-core. I should be able to get a few things done tonight since I don't have to wake up at the crack of dawn.